OTAnalytics/plugin_ui/nicegui_gui/pages/svz_metadata_form/svz_metadata_form.py

```diff
@@ -0,0 +1,163 @@
+from abc import ABC
+from typing import Self
+
+from nicegui import ui
+
+from OTAnalytics.adapter_ui.abstract_frame_project import AbstractFrameSvzMetadata
+from OTAnalytics.adapter_ui.view_model import ViewModel
+from OTAnalytics.application.project import (
+    COORDINATE_X,
+    COORDINATE_Y,
+    COUNTING_DAY,
+    COUNTING_LOCATION_NUMBER,
+    DIRECTION,
+    DIRECTION_DESCRIPTION,
+    HAS_BICYCLE_LANE,
+    IS_BICYCLE_COUNTING,
+    REMARK,
+    TK_NUMBER,
+    WEATHER,
+)
+from OTAnalytics.application.resources.resource_manager import (
+    ResourceManager,
+    SvzMetadataKeys,
+)
+
+
+class SVZMetadataForm(AbstractFrameSvzMetadata, ABC):
+
+    def __init__(self, viewmodel: ViewModel, resource_manager: ResourceManager) -> None:
+        self._viewmodel = viewmodel
+        self._resource_manager = resource_manager
+        self._directions = self._viewmodel.get_directions_of_stationing()
+        self._counting_day_types = self._viewmodel.get_counting_day_types()
+        self._weather_types = self._viewmodel.get_weather_types()
+        self.introduce_to_viewmodel()
+
+        self._tk_number: ui.input | None = None
```

Comment:
Please use FormFieldText for all ui.input elements, FormFieldCheckbox for all ui.checkbox elements and FormFieldSelect for all ui.select elements.
Set markers for all form fields